samples: dect_phy: dect_shell: new startup_cmd and possibility to give AT commands from command prompt (#18548)
nrfconnect/sdk-nrf: samples/dect/dect_phy/dect_shell/README.rst

      Stopping beacon.
      Beacon TX stopped, cause: User Initiated.

Running commands at bootup
==========================

DeSh command: ``startup_cmd``.

> You can use the ``startup_cmd`` command to store shell commands to be run sequentially after a bootup.

Reviewer: ```suggestion
You can use the ``startup_cmd`` command to store shell commands to be run sequentially after bootup.
```

Example: starting of cluster beacon and sending RA data to it
-------------------------------------------------------------

> * FT/Beacon device - Set a command to start a cluster beacon at 5 seconds after a bootup:

Reviewer: ```suggestion
* FT/Beacon device - Set a command to start a cluster beacon five seconds after bootup:
```

  .. code-block:: console

      desh:~$ startup_cmd -t 5 --mem_slot 1 --cmd_str "dect sett -t 1234"
      desh:~$ startup_cmd --mem_slot 2 --cmd_str "dect mac beacon_start"

> * PT/Client device - Set a command to start scanning for the beacon 60 seconds after a bootup, associate to it, and send RA data to it:

Reviewer: ```suggestion
* PT/Client device - Set a command to start scanning for the beacon 60 seconds after bootup, associate to it, and send RA data to it:
```
